modules/virt-planning-bare-metal-cluster-for-ocp-virt.adoc

```diff
@@ -5,17 +5,37 @@
 
 :_mod-docs-content-type: REFERENCE
 [id="virt-planning-bare-metal-cluster-for-ocp-virt_{context}"]
-= Planning a bare metal cluster for {VirtProductName}
+= Bare-metal cluster installation requirements for {VirtProductName}
 
-If you will use {VirtProductName}, it is important to be aware of several requirements before you install your bare metal cluster.
+[role="_abstract"]
+If you plan to use {VirtProductName} on a bare-metal cluster, you must ensure that your cluster is configured correctly during installation.
+This is because {VirtProductName} requires certain settings that cannot be changed after a cluster is installed.
 
-* If you want to use live migration features, you must have multiple worker nodes _at the time of cluster installation_. This is because live migration requires the cluster-level high availability (HA) flag to be set to true. The HA flag is set when a cluster is installed and cannot be changed afterwards. If there are fewer than two worker nodes defined when you install your cluster, the HA flag is set to false for the life of the cluster.
+[id="virt-planning-bare-metal-cluster-for-ocp-virt-HA_{context}"]
+== High availability requirements for {VirtProductName}
+
+When discussing high availability (HA) features in the context of {VirtProductName}, this refers only to the replication model of the core cluster components, determined by the `controlPlaneTopology` and `infrastructureTopology` fields. Setting these fields to `HighlyAvailable` offers component redundancy, which is distinct from general cluster-wide application HA. Setting these fields to `SingleReplica` disables component redundancy, and therefore disables {VirtProductName} HA features.
+
+If you plan to use {VirtProductName} HA features, you must have at least three control plane nodes at the time of cluster installation. The `controlPlaneTopology` status for the cluster must be `HighlyAvailable`.
```

Comment:
We should drop "at least" in the above line. 3 is the recommended count for control plane.